Bottom: 657dab6c52574e983fd79913eae7bf7e0320aa20
Top:    b45b6238003702e7cea33d22cbd4a38e309aa743
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:40:20 +0000

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 3b28cd9..18424ed 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -597,34 +597,25 @@ GList *
 object3d_from_board_outline (void)
 {
   POLYAREA *board_outline = board_outline_poly (true);
-
-#if 0
-  return object3d_from_soldermask_within_area (board_outline, TOP_SIDE);
-#else
-
   appearance *board_appearance;
   appearance *top_bot_appearance;
   GList *objects;
 
   board_appearance = make_appearance ();
-  top_bot_appearance = NULL;
-//  top_bot_appearance = make_appearance ();
+  top_bot_appearance = make_appearance ();
   appearance_set_color (board_appearance,   1.0, 1.0, 0.6);
-//  appearance_set_color (top_bot_appearance, 0.2, 0.8, 0.2);
+  appearance_set_color (top_bot_appearance, 0.2, 0.8, 0.2);
 
-#ifdef REVERSED_PCB_CONTOURS
   objects = object3d_from_contours (board_outline,
+#ifdef REVERSED_PCB_CONTOURS
                                     -HACK_BOARD_THICKNESS, /* Bottom */
                                     0                    ,  /* Top */
-                                    board_appearance,
-                                    top_bot_appearance);
 #else
-  objects = object3d_from_contours (board_outline,
                                      HACK_BOARD_THICKNESS / 2, /* Bottom */
                                     -HACK_BOARD_THICKNESS / 2, /* Top */
+#endif
                                     board_appearance,
                                     top_bot_appearance);
-#endif
 
   destroy_appearance (board_appearance);
   destroy_appearance (top_bot_appearance);
